lib/taskana-cdi/src/test/java/pro/taskana/TaskanaProducersTest.java

```diff
@@ -33,19 +31,13 @@
         deployment.addAllDependencies();
         deployment.addDependency("org.mybatis:mybatis:3.4.2");
         deployment.addDependency("org.mybatis:mybatis-cdi:1.0.0");
-        deployment.addDependency("pro.taskana:taskana-core:1.1.4-SNAPSHOT");
+        deployment.addDependency("pro.taskana:taskana-core");
```

Comment:
What does this change exactly do? Will the latest version of taskana-core be used?
If so, then please remove the -swarm parameter from the change_version call in .travis.yml

Reply:
It will use the Version specified in the pom. So the latest version will be used.